docs/DeveloperGuide.md

```diff
@@ -309,28 +309,71 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 (For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
-
-**MSS**
-
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
-
-    Use case ends.
-
-**Extensions**
-
-* 2a. The list is empty.
-
-  Use case ends.
-
-* 3a. The given index is invalid.
-
-    * 3a1. AddressBook shows an error message.
-
-      Use case resumes at step 2.
+**System: ArtBuddy (AB)**<br>
```

Comment:
Since we have the line above (line 310) specifying the system and actor for all cases, perhaps we can omit these two lines and update the above `user` with `Artist`

Reply:
Yep, makes sense- changed it accordingly!